app/src/main/java/org/mozilla/fenix/components/appstate/AppState.kt

```diff
@@ -41,6 +41,7 @@ import org.mozilla.fenix.gleanplumb.MessagingState
  */
 data class AppState(
     val inactiveTabsExpanded: Boolean = false,
+    val inactiveTabsAutoCloseDialogShown: Boolean = false,
```

Comment:
I see there is also a `TabTrayStore`. Can you investigate why `inactiveTabsExpanded` was originally in the `HomeFragmentStore` and whether or not both of these **should** go into the `TabTrayStore`? I don't expect to see the refactoring of `inactiveTabsExpanded ` to be done in this PR.

Reply:
Ah seems like `inactiveTabsExpanded` was always in `AppState`

Reply:
`inactiveTabsAutoCloseDialogShown` actually isn't used. It's leftover from something else. Sorry for the confusion.